docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,71 @@
+# Release 465 (15 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in [EWKB
+  format](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry)
+  with the {func}`ST_GeomFromBinary` function. ({issue}`23824`)
+* Add support for the `ORDER BY` clause in a windowed aggregate function. ({issue}`23929`)
+* Improve performance of queries with selective joins on large build sides. ({issue}`22824`)
+* Fix potential configuration leakage between catalogs using JDBC-based
+  connectors. ({issue}`24058`)
+* Reduce chance of out of memory query failure when `retry-policy` is set to
+  `task`. ({issue}`24114`)
+
+## CLI
+
+* Fix incorrect quoting of output values when the `CSV_UNQUOTED` or the 
+  `CSV_HEADER_UNQUOTED` format is used. ({issue}`24113`)
+
+## BigQuery connector 
+
+* Fix failure when reading views with `timestamp` columns. ({issue}`24004`)
+
+## Cassandra connector
+
+* {{breaking}} Require setting `cassandra.security=PASSWORD` along with
+  `cassandra.username` and `cassandra.password` for  password-based
+  authentication. ({issue}`23899`)
+
+## Clickhouse connector
+
+* Fix insert for tables with the Clickhouse `TimestampWithTimeZone` type. ({issue}`23785`)
```

Comment:
What was fixed? How was it broken? (Incorrect results, query failures, etc).

Reply:
@ssheikin @Praveen2112 .. not sure how it was broken before from looking the the PR ... did it actually fail or did it use to insert incorrect data?

Reply:
If a row has a timestamp with different zone when compared with the zone specified as a part of column - then it is not normalized when inserting into clickhouse.

Reply:
So it looks like it inserted wrong data in the past .. correct?

Reply:
Yes